manywheel/build_libtorch.sh

```diff
@@ -23,10 +23,10 @@ retry () {
 # TODO move this into the Docker images
 OS_NAME=`awk -F= '/^NAME/{print $2}' /etc/os-release`
 if [[ "$OS_NAME" == *"CentOS Linux"* ]]; then
-    retry yum install -q -y zip openssl openssl-devel
+    retry yum install -q -y zip openssl
```

Comment:
this is a rollback of https://github.com/pytorch/builder/pull/623 because it install 1.0.2